prisma/schema.prisma

```diff
@@ -1845,8 +1845,12 @@ model HomeBlock {
   userId    Int
   user      User          @relation(fields: [userId], references: [id], onDelete: Cascade)
   metadata  Json          @default("{}")
-  index     Int
+  index     Int?
```

Comment:
I still think index should be required, wouldn't it? Given that we are going to let people place homeblocks in any order they want

Reply:
That's why it can't be. We don't know the index position once the user initially adds to their home. So we need nullsLast